support/troubleshooting/investigating-monitoring-issues.adoc

```diff
@@ -0,0 +1,21 @@
+[id="investigating-monitoring-issues"]
+= Investigating monitoring issues
+include::modules/common-attributes.adoc[]
+:context: investigating-monitoring-issues
+
+toc::[]
+
```

Comment:
You need an intro sentence.

Reply:
@kalexand-rh thank you for your review. Can you please take a look at the intro that I have now added? Thanks!

Reply:
One point to note is that the second paragraph in the intro would need to be updated when we add further modules to this page. But, at present I don't feel that there are enough modules in this assembly to say, "You can follow these procedures if you experience issues with the monitoring stack".

Reply:
I think that's fine! I suggest adding a comment to remind whoever might add the next module to this assembly to also update that sentence.

Reply:
That's a great idea, thank you. Resolved.